pages/linux/opusenc.md

```diff
@@ -0,0 +1,20 @@
+# opusenc
+
+> Convert WAV or FLAC audio to Opus.
+> More information: <https://opus-codec.org/docs/opus-tools/opusenc.html.
+
+- Convert WAV to Opus using default options:
+
+`opusenc {{input.wav}} {{output}}.opus`
```

Comment:
We use path to files in the examples.
```suggestion
`opusenc {{path/to/input.wav}} {{path/to/output.opus}}`
```

Reply:
I'm slightly confused by this comment. I followed the example of `ffmpeg` and `convert. Both assume the file is in the current working directory.  Should this be changed?

Reply:
:thinking: sometimes we have exceptions to the rules. @owenvoke can you help?

Reply:
Hmm. I think in general, we always try to use the full path syntax (guidelines 4 and 5 in the token syntax section of the style guide) except if:
- a. the file location is implicit
- b. the command example would be too long (for example, the last command in the `ffmpeg` page).

I'd say this might meet the first exemption, so this should be fine, but I'll defer to @sbrl as they have more experience. 👍🏻 I'd probably put it as the longer path, although the 2nd example perhaps looks a bit odd.

![Screenshot 2020-10-05 at 12 38 10](https://user-images.githubusercontent.com/1899334/95075152-a3d8ec00-0707-11eb-97a7-dcd49587e098.png)